ext/standard/basic_functions.c

```diff
@@ -652,15 +654,11 @@ PHP_FUNCTION(long2ip)
 	ip = (zend_ulong)sip;
 
 	myaddr.s_addr = htonl(ip);
-#ifdef HAVE_INET_PTON
 	if (inet_ntop(AF_INET, &myaddr, str, sizeof(str))) {
 		RETURN_STRING(str);
 	} else {
 		RETURN_FALSE;
 	}
```

Comment:
Can this actually fail? because otherwise the call to this in dns.c looks fishy?

Reply:
I would need to recheck some of them indeed.